python_versions_check.py

```diff
@@ -31,17 +32,22 @@
 def run(koji_build, workdir='.', artifactsdir='artifacts',
         testcase='dist.python-versions', arches=['x86_64', 'noarch', 'src']):
     '''The main method to run from Taskotron'''
-    workdir = os.path.abspath(workdir)
-    results_path = os.path.join(artifactsdir, 'taskotron', 'results.yml')
-    artifact = os.path.join(artifactsdir, 'output.log')
+    artifactsdir = pathlib.Path(artifactsdir)
+    workdir = pathlib.Path(workdir).resolve()
+    resultsdir = artifactsdir / 'taskotron'
+    resultspath = resultsdir / 'results.yml'
+    artifact = artifactsdir / 'output.log'
+
+    artifactsdir.mkdir(parents=True, exist_ok=True)
+    resultsdir.mkdir(parents=True, exist_ok=True)
```

Comment:
Those two mkdirs were needed.
cc @kparal and @frantisekz - previously it worked without this